Javascript(ES5)/README.md

```diff
@@ -733,6 +1098,39 @@ Forked from the excellent [Airbnb JavaScript Style Guide](https://github.com/air
       return type;
     }
     ```
+  - Start all comments with a space to make it easier to read.
```

Comment:
And ironically enough, that list item does not have a space above it, lol. (I'll fix it.)